conf/machine/colibri-vf.conf

```diff
@@ -35,6 +35,7 @@ PREFERRED_PROVIDER_u-boot-default-script ?= "u-boot-script-toradex"
 PREFERRED_PROVIDER_virtual/kernel-module-mcc ?= "kernel-module-mcc-toradex"
 PREFERRED_PROVIDER_virtual/kernel-module-mcc-dev ?= "kernel-module-mcc-toradex"
 PREFERRED_VERSION_mqxboot ?= "1.%"
+PREFERRED_VERSION_weston_vf = ""
```

Comment:
Use _use-nxp-bsp for the override as mainline would work.

Reply:
Ok.